src/dataflow/components/nodes/controls/plot-button-control.tsx

```diff
@@ -45,7 +45,7 @@ export class PlotButtonControl extends Rete.Control {
   public setGraph = (show: boolean) => {
     this.props.showgraph = show;
     this.putData(this.key, show);
-    (this as any).update();
+    this.node.update();
```

Comment:
I'd recommend a comment here to explain why we're doing this. I removed it as part of my earlier work because no other controls do this and when I tried taking it out it seemed like everything was working.